kuka_kr270_support/package.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>kuka_kr270_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the KUKA KR 270 (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for KUKA KR 270 manipulators. This currently includes the R2700 only.
+    </p>
+    <p><b>Specifications</b>:</p>
+    <ul>
+      <li>KR 270 R 2700 - Default</li>
+    </ul>
+    <p>
+      Joint limits and maximum joint velocities are based on the information
+      found in the online datasheet of the Kuka KR 270 R2700 ultra.
+      <!-- <a href="https://www.kuka.com/-/media/kuka-downloads/imported/6b77eecacfe542d3b736af377562ecaa/0000181028_de.pdf?rev=ec8dfd6ea34947688ab70f343d481a7e&hash=01A6578B4726370C788A35703EE5036D">datasheet</a>. -->
+      <!-- KUKA KR 270 R2700 ultra - 0000181028_DE.pdf, version 0000-181-028 / V46.1 / 07.01.2021 / de -->
+      All urdfs are based on the default motion and joint velocity limits,
+      unless noted otherwise.
+    </p>
+    <p>
+      Before using any of the configuration files and / or meshes included
+      in this package, be sure to check they are correct for the particular
+      robot model and configuration you intend to use them with.
+    </p>
+    <p>
+      <b>NB</b>: this package currently uses non-valid inertia parameters.
+    </p>
+  </description>
+  <author email="jan.krieglstein@ipa.fraunhofer.de">Jan Krieglstein (Fraunhofer IPA)</author>
```

Comment:
We don't typically include the `email` attribute for `author`, but we can keep it if you'd like.

Reply:
Done in 23dfcdafc865e13b7f0d0d05735b48e0dd8f5287